store/firstlast.go

```diff
@@ -1,41 +0,0 @@
-package store
-
-import (
-	"bytes"
-
-	sdk "github.com/cosmos/cosmos-sdk/types"
-	sdkkv "github.com/cosmos/cosmos-sdk/types/kv"
-)
-
-// First gets the first item.
-func First(st KVStore, start, end []byte) (kv sdkkv.Pair, ok bool) {
```

Comment:
Have you verified in https://cs.github.com than no one depends on that?
We should add a chsngelog too :)

Reply:
I'll be checking all of them in that search engine.

Using the query [language:go ".First(" "cosmos-sdk/store"](https://cs.github.com/?scopeName=All+repos&scope=&q=language%3Ago+%22.First%28%22+%22cosmos-sdk%2Fstore%22) doesn't return any results

Using the query [language:go ".Last(" "cosmos-sdk/store"](https://cs.github.com/?scopeName=All+repos&scope=&q=language%3Ago+%22.Last%28%22+%22cosmos-sdk%2Fstore%22) returns only some **unrelated** results